### PR TITLE
QuickOpen improvements

### PR DESCRIPTION
**Pull request by stijn-h**
This commit does the following:

- Removed code that would add directories to the search results. It wasn't used anywhere, and when turned on selecting a directory would throw an error. It doesn't seem desirable, so I think we should remove it.
- Made the popup window slightly smaller.
- Some small code improvements. No logic is changed or added.

### CODE REVIEW

File: editor/quick_open.cpp [hunk 1]
```diff
@@ -29,24 +29,20 @@
 /*************************************************************************/
 
 #include "quick_open.h"
-
```
Comment (akien-mga) [inline editor/quick_open.cpp:32]: This should not be removed. https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

**Review by akien-mga — APPROVED** [review]
Left one style comment, but looks good otherwise.

**Comment by DarkMessiah** [comment]
I think it would be nice to add ability (checkbox or something) to exclude addons folder from search in QuickOpen scene and script.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.3.

**Pull request merged**